src/components/ProjectSafeDashboard/TransactionCollapse.tsx

```diff
@@ -40,10 +40,11 @@ export function TransactionCollapse({
   const [expanded, setExpanded] = useState<boolean>(Boolean(selected))
 
   let versionText: string | undefined
-  switch (version) {
-    case CV_V1:
-      versionText = 'V1'
-      break
+  switch (cv) {
+    // TODO why do we need '1' here on V2V3 component?
```

Comment:
I think this component is version agnostic, used in safe page for both v1 and v2

Reply:
oh wait, i could have sworn this was a V2V3 component... nvm